dispatcher/MongoConnect.py

```diff
@@ -636,7 +636,8 @@ def detector_ackd_command(self, detector, command):
         return True
 
     def log_error(self, message, priority, etype):
-
+        #Start by logging the error localy
+        self.log.INFO(f"{message}")
```

Comment:
Is the method `.INFO` or `.info`? I thought the functions were lowercase and the log-level enums were uppercase. Also, is the fstring strictly necessary? That should have already been done on the other end of this function call.

Reply:
You are correct in both things